pwndbg/commands/start.py

```diff
@@ -110,9 +96,7 @@ def start(args=None) -> None:
 @pwndbg.commands.ArgparsedCommand(parser, category=CommandCategory.START)
 @pwndbg.commands.OnlyWithFile
 def entry(args=[]) -> None:
-    global break_on_first_instruction
-    break_on_first_instruction = True
-    run = "run " + " ".join(map(quote, args))
+    run = "starti " + " ".join(map(quote, args))
```

Comment:
I believe this is wrong: see the documentation for this command:
```
Start the debugged program stopping at its entrypoint address.
Note that the entrypoint may not be the first instruction executed
by the program. If you want to stop on the first executed instruction,
use the GDB's `starti` command.
(...)
```
`starti` starts at very first instruction the process will execute - it can be the entrypoint of ld.so, while `entry` stops at the entrypoint of the binary itself.